index.js

```diff
@@ -47,6 +46,9 @@ class Installer {
         }
       )
     }).then(() => {
+      return config(this.prefix, process.argv, this.pkg)
```

Comment:
If we can avoid needing `this.pkg` for the UA, then this can be done as part of the previous join in parallel to reading package.json etc.

Reply:
Never mind ^

Reply:
Eventually this won't be slow so it won't matter anyway, and if it does matter, we can address later.